### ISSUE TITLE
HBBTV-Formate shows wrong resolution

### ISSUE THREAD

**Issue by ismail** [rols1/Kodi-Addon-ARDundZDF#7]
First of all thank you for this add-on, it's great to have access to all these content right from Kodi.

Now to the bug report. This happens with all the HBBTV links I tried so far (Bergretter, Traumschiff, etc.) but to give a concrete example: If you go to 

`ZDF Mediathek -> Meist gesehen (1 Woche) -> heute show: heute-show vom 9. April 2021 -> HBBTV-Formate`

```
[...]
2. MP4, Qualität: FULL HD | Normal | Bitrate 3360 | Auflösung 1920x1080
[...]
4. HLS, Qualität: FULL HD | Normal | Bitrate 6501324 | Auflösung 1920x1080
```

Now click either 2 or 4 and select `Settings -> Video settings` from Kodi menu below. You'll see that resolution is only `1280x720`

Tested on Kodi 19 on an Android TV (v9)

**Comment by rols1** [comment]
thanks for your report. Indeed, the details in addon about bitrate and resolution are wrong. More precisely: I have taken random samples. Some users reported about some broadcasts in full hd - possibly a testing phase of ZDF.  But I have never seen the FULL HD value 1920x1080 myself in HBBTV-sources.
The reason for random samples: only the HLS-Streams contain bandwith and resolution. But it needs to download the m3u8-file to get them. And the mp4-links only shows the bitrate.

Now I've corrected the values to the common values. [Testaddon on gmx](https://c.gmx.net/@327723216099344844/cASZJUNuQpqlEImIS62dmA), Update expected to weekend.

If you are interested in details:
here the hbbtv-links for a Traumschiff-broadcast, [source](https://hbbtv.zdf.de/zdfm3/dyn/get.php?id=SCMS_d7e7b0bc-6cb4-4f64-939a-70a8654e3a05):
`"q1":"http://tvdlzdf-a.akamaihd.net/none/zdf/21/04/210414_1631_hko/1/210414_1631_hko_2360k_p35v15.mp4"
"q3":"http://tvdlzdf-a.akamaihd.net/none/zdf/21/04/210414_1631_hko/1/210414_1631_hko_3360k_p36v15.mp4"
"q1":"https://zdfvodnone-vh.akamaihd.net/i/meta-files/zdf/smil/m3u8/300/21/04/210414_1631_hko/1/210414_1631_hko.smil/master.m3u8"
"q3":"https://zdfvodnone-vh.akamaihd.net/i/meta-files/zdf/smil/m3u8/veryhigh/21/04/210414_1631_hko/1/210414_1631_hko.smil/master.m3u8"`

/R

**Comment by ismail** [comment]
Thanks! I will have to wait for an official update to test it though, since I don't have a PC atm.

**Comment by ismail** [comment]
Confirmed fixed in 3.8.1, cheers!
